advanced/index.php

```diff
@@ -102,8 +102,10 @@ function setHeader($type = "x") {
 $bpAskAdmin = !empty($svEmail) ? '<a href="mailto:'.$svEmail.'?subject=Site Blocked: '.$serverName.'"></a>' : "<span/>";
 
 // Determine if at least one block list has been generated
-if (empty(glob("/etc/pihole/list.0.*.domains")))
+$blocklistglob = glob("/etc/pihole/list.0.*.domains");
+if ($blocklistglob = "") {
```

Comment:
`$blocklistglob = ""` will set `$blocklistglob` to `""`. This will always succeed hence the condition is always fulfilled. I guess you want `==` or (even better) `===`. Note that with `===` you will have to test for the exact match, so e.g. `if($blocklistglob === array())` to test against an empty array.